UnicornRender/video/source/vulkan/Renderer.cpp

```diff
@@ -1503,9 +1493,9 @@ bool Renderer::AllocateMaterial(const Mesh& mesh, VkMesh& vkmesh)
 
             auto result = m_vkLogicalDevice.allocateDescriptorSets(&allocInfo, &descriptorSet);
 
-            if(result != vk::Result::eSuccess)
+            if(result != vk::Result::eSuccess || !vkTexture->IsInitialized())
             {
-                LOG_ERROR("Can't allocate descriptor sets!");
+                LOG_ERROR("Can't allocate sampler descriptor sets!");
```

Comment:
Well it's not technically true - it either can't allocate descriptor sets or the texture is not initialized. Does it make sense to attempt to allocate descriptor sets if texture is not initialized?